config/application.yml.example

```diff
@@ -4,3 +4,4 @@ acr_values: 'http://idmanagement.gov/ns/assurance/loa/1'
 redirect_uri: 'http://localhost:9292/'
 client_id: 'urn:gov:gsa:openidconnect:sp:sinatra'
 sp_private_key_path: 'config/demo_sp.key'
+cache_oidc_config: True
```

Comment:
why the capital `True`?

Reply:
ahhh, old habit. Both parse to `true`, but will submit PR to fix